src/plumbing/fnk/impl.clj

```diff
@@ -37,6 +37,16 @@
   [k]
   (symbol (name k)))
 
+(defn qualified-k->sym
+  "Make a keyword into a symbol, preserving namespace"
+  [k]
+  (symbol (namespace k) (name k)))
```

Comment:
Err, i see k->sym already exists above.  That can just be fixed to work on qualified symbols rather than adding a new fn i think.